@xen-orchestra/lite/src/composables/collection-sorter.composable.md

```diff
@@ -0,0 +1,41 @@
+# useCollectionSorter composable
+
+## Usage
+
+```typescript
+const { sorts, addSort, removeSort, compareFn, toggleSortDirection } =
+  useCollectionSorter(options);
+
+const sortedCollection = computed(() => myCollection.sort(compareFn));
+addSort("name", true);
+addSort("age", false);
+```
+
+## Options
+
+### `queryStringParam`
+
+This option allows to activate the URL Query String support.
+
+```typescript
+const { addSort } = useCollectionSorter({ queryStringParam: "sort" });
+addSort("name", true); // Will update the URL with ?sort=name:1
+```
+
+### Initial sorts
+
+This option allows to set some initial sorts.
+
+```typescript
+const {
+  /* ... */
+} = useCollectionSorter({
+  initialSorts: [
+    ["name", true],
+    ["age", false],
+  ],
+});
```

Comment:
I wonder if that's clear enough.
In terms of readability, it seems strange to me to use `true/false` to define a sort order.
No better to handle `asc/desc` and ignore all other string values?